### ISSUE TITLE
Not wokring on Ionic 5.0.2 

### ISSUE THREAD

**Issue by ranafeb14** [HsuanXyz/ionic-stepper#27]
Its failing on ionic 5.0.2 initially failed with 

ionic-angular import changed the import to @ionic/angular.

Now failing with below error

```
Uncaught Error: Template parse errors:
Can't bind to 'ngIf' since it isn't a known property of 'div'. ("                         [active]="i <= selectedIndex">
          </ion-step-header>
          <div [ERROR ->]*ngIf="!isLast" class="ionic-stepper-horizontal-line"></div>
      </ng-container>
    </div>
"): ng:///IonicStepperModule/IonicStepperComponent.html@11:15
Property binding ngIf not used by any directive on an embedded template. Make sure that the property name is spelled correctly and all directives are listed in the "@NgModule.declarations". ("
                             [active]="i <= selectedIndex">
[ionic-stepper-5.0.2-error.log](https://github.com/hsuanxyz/ionic-stepper/files/3316969/ionic-stepper-5.0.2-error.log)

```

**Comment by Et3rnal** [comment]
any luck with this?

**Comment by astidesai** [comment]
I have the same issue

**Comment by pauloXavier89** [comment]
same issue here. Did anyone have success?

**Comment by PabloDSL** [comment]
Seems like your error is not related to the steeper, instead is a error about the *ngIf directive. Check if you have that component declared in their respective parent module, if yes, then check if '@angular/common' is imported

**Comment by SerginhuXavier** [comment]
> Seems like your error is not related to the steeper, instead is a error about the *ngIf directive. Check if you have that component declared in their respective parent module, if yes, then check if '@angular/common' is imported

Pablo, the error occurs inside the module. It's because in angular 4 the sintax of core imports is "angular/" and in Angular 5 is "@angular/".

**Comment by pauloXavier89** [comment]
> Seems like your error is not related to the steeper, instead is a error about the *ngIf directive. Check if you have that component declared in their respective parent module, if yes, then check if '@angular/common' is imported

Pablo, the error occurs inside the module. It's because in angular 4 the sintax of core imports is "angular/" and in Angular 5 is "@angular/".

**Comment by PabloDSL** [comment]
> > Seems like your error is not related to the steeper, instead is a error about the *ngIf directive. Check if you have that component declared in their respective parent module, if yes, then check if '@angular/common' is imported
> 
> Pablo, the error occurs inside the module. It's because in angular 4 the sintax of core imports is "angular/" and in Angular 5 is "@angular/".

You're right, it's a problem about the new import sintax inside the module itself, my bad. I tried updating the imports by hand but with no success.
at the end I just decided to not use it and use ionic slides instead

**Comment by anreig** [comment]
Hello.
Here you have support for ionic 5: https://github.com/anreig/ionic-stepper

Thanks

**Comment by magiclockers** [comment]
not working in angular 9+
the ivy renderer won't load this library any more :(
新的ivy渲染器無法加載這個模塊了，新的angular沒法使用了😯
